homeassistant/components/config/config_entries.py

```diff
@@ -409,13 +432,13 @@ async def ignore_config_flow(
 
 @websocket_api.websocket_command(
     {
-        vol.Required("type"): "config_entries/get",
+        vol.Required("type"): "config_entries/get_matching",
```

Comment:
This breaks both the insteon panel and the knx panel since we are using this command to get the installed config entries via ha-frontend, does this need to be a breaking change?

Reply:
Is it still a problem with the latest frontend version?

Reply:
https://github.com/home-assistant/core/pull/93581

Reply:
Yes, the problem still persists for Insteon, KNX and Dynalite at least since we have to import a frontend tag as git Submodule to be able to use HA components in an iFrame. So we have to update our upstream packages - which isn't that straight forward since there are so many breaking changes made in Frontend - and tbh it is not not very well documented...

I can imagine some custom frontend cards will have the same issue, but I don't know much about those.

Reply:
What @farmio said is valid. The error is still there. But to come back to the original question:

Why was it neccessary to rename an existing websocket command instead of simply creating, for instance, a new `config_entries/get_single` command that would not break existing functionality?

And to add to that, would a change that does exactly this be accepted in a follow-up PR because I doubt everyone will update there HA Frontend tag in those custom panels to the latest version until next release. Not even thinking about all the custom frontend adjustments that people who rely on this functionality have to make.

Reply:
Cc @teharris1 @ziv1234

Reply:
I just had a chat with @emontnemery and we decided to partially rewrite this change to be non-breaking as also other custom frontend integrations might be affected and not all panels are updated already.